clash-lib/src/Clash/Normalize/Transformations/DEC.hs

```diff
@@ -1,6 +1,6 @@
 {-|
   Copyright  :  (C) 2015-2016, University of Twente,
-                    2021,      QBayLogic B.V.
+                    2022,      QBayLogic B.V.
```

Comment:
Don't you mean

```
2021,      QBayLogic B.V.
2022,      LumiGuide Fietsdetectie B.V.
```

Reply:
It feels almost dirty adding this line but Oke :p

Reply:
It was a team effort :heart:

Reply:
But you did the actual typing